serializer.rst

```diff
@@ -368,6 +368,83 @@ stored in one of the following locations:
 
 .. _serializer-enabling-metadata-cache:
 
+Using nested attributes
+-----------------------
+
+To map nested properties, a ``SerializedPath`` can be defined with annotations, 
+attributes and YAML or XML configurations:
+
+.. configuration-block::
+
+    .. code-block:: php-annotations
+
+        namespace App\Model;
+
+        use Symfony\Component\Serializer\Annotation\SerializedPath;
+
+        class Person
+        {
+            /**
+             * @SerializedPath("[profile][information][birthday]")
+             */
+            private $dob;
```

Comment:
I would call it $birthday to mimic the getBirthday below 
even better, use for ex a `private string $name` to be more precise than just a date in string format :)